modules/ROOT/pages/cache.adoc

```diff
@@ -0,0 +1,25 @@
+= Optionally Configure the Cache
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+Anypoint API Community Manager can improve performance with a platform cache.
```

Comment:
You can improve the performance of pages and components in your API community by using the Platform Cache [LINK TO https://help.salesforce.com/articleView?id=data_platform_cache.htm&type=5]

Reply:
Fixed.